app/test_helpers.go

```diff
@@ -273,7 +273,6 @@ func IntegrationTestNetworkConfig() network.Config {
 	cfg.LegacyAmino = encCfg.Amino
 	cfg.InterfaceRegistry = encCfg.InterfaceRegistry
 	cfg.GenesisState = appGenState
-	cfg.BondDenom = params.BondDenom
```

Comment:
This is an interesting bit that's in [umee's helpers](https://github.com/umee-network/umee/blob/36171b24fbb225e2df1b1116348b9e56d3911b57/app/test_helpers.go#L316), but breaks on the most recent version of the SDK. 

This is likely not an issue there bc Umee is on an SDK fork

Reply:
jk! this was an issue because I forgot to override the default genesis for a few modules. got it now